src/pages/public/Register.tsx

```diff
@@ -0,0 +1,368 @@
+          ].map(([key, label]) => (
+            <TextField
+              key={key}
+              label={label}
+              className="mb-4 text-brand"
+              fullWidth
+              onChange={(e) => props.setFieldValue(key, e.currentTarget.value)}
+            />
+          ))}
+        </>
+      )
+    },
+    {
+      stepIndex: 1,
+      Content: () => (
+        <>
+          <FormTitle as="h2" className="mb-4">
+            Foto deines Hofes
+          </FormTitle>
+          <UploadFarmPhotoSvg className="mx-auto" />
+        </>
+      )
+    }
+  ],
+  [
+    {
+      stepIndex: 2,
+      Content: () => (
+        <>
+          <div className="mb-8">
+            <FormTitle as="h2" className="mb-2">
+              Mindestbeschäftigungsdauer
+            </FormTitle>
+            <div className="flex">
+              {/* TODO: switch to radio buttons, when implementing form */}
+              {/* FIXME: first-child selector not working. god knows why... */}
+              {["1/2 Tag", "1 Tag", "1 Woche", "Länger"].map((text, i) => (
+                <button
+                  key={i}
+                  className="flex-grow border-2 border-r-0 last:border-r-2 first:rounded-l-full last:rounded-r-full border-brand text-brand text-sm py-1 px-2 first:pl-4 last:pr-4 font-medium focus:text-white focus:bg-brand"
+                >
+                  {text}
+                </button>
+              ))}
+            </div>
+          </div>
+          <div className="mb-8">
+            <FormTitle as="h2" className="mb-2">
+              Stundenlohn
+            </FormTitle>
+
+            <StepCalculator
+              initialValue={9.35}
+              steps={0.5}
+              min={0}
+              renderValue={(v) => `${v.toFixed(2)} €`}
+            />
+          </div>
+          <div className="mb-8">
+            <FormTitle as="h2" className="mb-2">
+              Übernachtungsmöglichkeiten für Helfer*innen
+            </FormTitle>
+            <div>
+              <Radio block>Ja</Radio>
+              <Radio block>Nein</Radio>
+            </div>
+          </div>
+          <div>
+            <FormTitle as="h2" className="mb-2">
+              Übernachtungspreis
+            </FormTitle>
+            <StepCalculator
+              initialValue={5}
+              steps={0.5}
+              min={0}
+              renderValue={(v) => `${v.toFixed(2)} €`}
+            />
+          </div>
+        </>
+      )
+    },
+    {
+      stepIndex: 2,
+      Content: () => (
+        <>
+          <div className="mb-8">
+            <FormTitle as="h2" className="mb-2">
+              Anreise der Helfer*innen
+            </FormTitle>
+            <div>
+              <Radio block>Eigenständig (keine Abholung)</Radio>
+              <Radio block>
+                Abholung möglich, mit einer maximalen Entfernung von bis zu ...
+              </Radio>
+            </div>
+            <StepCalculator
```

Comment:
Wie würden diese Werte in der Form landen?
Und können wir den StepCalculator erst aktivieren wenn die "Abholung möglich" Option ausgewählt wird?

Reply:
Werte werden im Form-State hinterlegt. Die neuen Updates sollten selbsterklärend sein. StepCalculate wird nun dynamisch gerendert